rosdep/python.yaml

```diff
@@ -2516,6 +2516,37 @@ python-pynmea2:
   ubuntu:
     pip:
       packages: [pynmea2]
+python-pypng:
+  arch:
+    pip:
+      packages: [pypng]
+  debian: [python-png]
+  fedora: [python-pypng]
+  gentoo: [dev-python/pypng]
+  ubuntu:
+    artful: [python-png]
+    bionic: [python-png]
+    trusty:
+      pip:
+        packages: [pypng]
+    utopic:
```

Comment:
We usually don add new keys for EOL distributions. Can you please keep only trusty and xenial here?

For future proofness, we can use the new wildcard syntax to cover upcoming distributions as wel like cosmic:
```
ubuntu:
  '*': [python-png]
  trusty:
    pip:
      packages: [pypng]
  xenial:
    pip:
      packages: [pypng]
```